extensions/cursor-directory/src/index.tsx

```diff
@@ -66,111 +74,108 @@ export default function Command() {
         </List.Dropdown>
       }
     >
+      {isLoadingStarredRules ? (
+        <List.EmptyView title="Loading starred cursor rules..." />
+      ) : (
+        starredRulesData &&
+        starredRulesData.length > 0 && (
+          <List.Section title="Starred Cursor Rules">
+            {starredRulesData.map((data) => {
+              const cursorRule = allRules.find((item) => item.slug === data);
+
+              if (!cursorRule) return null;
+
+              const props = showingDetail
+                ? {
+                    detail: <CursorRulePreview cursorRule={cursorRule} popularOnly={popularOnly} />,
+                  }
+                : {
+                    accessories: [
+                      { text: cursorRule.tags.slice(0, 3).join(", ") },
+                      ...(popularOnly && cursorRule.count !== null
+                        ? [{ icon: Icon.Person, text: cursorRule.count.toString() }]
+                        : []),
+                      { icon: { source: Icon.Star, tintColor: Color.Yellow } },
+                    ],
+                  };
+
+              return (
+                <List.Item
+                  key={cursorRule.slug}
+                  title={cursorRule.title}
+                  {...props}
+                  actions={
+                    <ActionPanel>
+                      <ActionPanel.Section title="Actions">
+                        <ViewRuleAction cursorRule={cursorRule} popularOnly={popularOnly} />
+                        <CopyRuleAction cursorRule={cursorRule} />
+                        <ToggleViewAction showingDetail={showingDetail} setShowingDetail={setShowingDetail} />
+                        <Action
+                          title="Unstar Cursor Rule"
+                          icon={Icon.StarDisabled}
+                          shortcut={{ modifiers: ["cmd"], key: "u" }}
```

Comment:
In Raycast we use the same hotkey for both unpin/pin - you can use `Keyboard.Shortcut.Common.Pin`

Reply:
Thanks for pointing!